code/game/gamemodes/objectives_team.dm

```diff
@@ -0,0 +1,24 @@
+//A barebones antagonist team.
+/datum/objective_team
+	var/list/datum/mind/members = list()
+	var/name = "team"
+	var/member_name = "member"
+
+/datum/objective_team/New(starting_members)
+	. = ..()
+	if(starting_members)
+		if(islist(starting_members))
+			for(var/datum/mind/M in starting_members)
+				add_member(M)
+		else
+			add_member(starting_members)
+		members += starting_members
```

Comment:
I think I could just be very confused here cause I don't really know what |= does for lists (i'm assuming append?) but it seems like the starting members were already added at this point so what's the point of doing += again here?

Reply:
|= is append... read this ass add this player to the list(add_member), THEN when we are done, add all of that to the members list. so no it wasn't

Reply:
Okay, and += combines lists right? What if starting_members is only one mind and not a list here? What would += do?

Also doesn't `add_member` already directly add to the `members` list?

Reply:
it adds the one person to the list but i will go look at it again later.

Reply:
okay yeah will fix later